### PR TITLE
feat: sample buckets 3 for usdc to any token dashboard 

### PR DESCRIPTION
**Pull request by lynnshaoyu**

### CODE REVIEW

**Review by mikeki — CHANGES_REQUESTED** [review]
Having that many buckets is a little bit counterproductive, we have 22~35% of requests for amounts lower than 750, if we divide them in this many buckets we will increase the number of cache misses.

File: lib/handlers/router-entities/route-caching/cached-routes-configuration.ts [hunk 1]
```diff
@@ -159,9 +159,12 @@ export const CACHED_ROUTES_CONFIGURATION: Map<string, CachedRoutesStrategy> = ne
       tradeType: TradeType.EXACT_INPUT,
       chainId: ChainId.MAINNET,
       buckets: [
-        new CachedRoutesBucket({ bucket: 100, cacheMode: CacheMode.Tapcompare, withLastNCachedRoutes: 10 }),
```
Comment (mikeki) [inline lib/handlers/router-entities/route-caching/cached-routes-configuration.ts:162]: Let's instead try to just Darkmode this bucket for 100 and see how things change

**Review by mikeki — APPROVED** [review]

**Comment by jsy1218** [comment]
There's a test failure, seems like test update needed

**Comment by jsy1218** [comment]
> Having that many buckets is a little bit counterproductive, we have 22~35% of requests for amounts lower than 750, if we divide them in this many buckets we will increase the number of cache misses.

I wonder if we have a way to canary changeset like this?

**Pull request merged**